src/Microsoft.AspNetCore.Sockets.Http/SocketsAppBuilderExtensions.cs

```diff
@@ -18,7 +18,10 @@ public static IApplicationBuilder UseSockets(this IApplicationBuilder app, Actio
 
             callback(new SocketRouteBuilder(routes, dispatcher));
 
-            app.UseWebSockets();
+            app.UseWebSockets(new WebSocketOptions()
+            {
+                ReceiveBufferSize = 14
```

Comment:
Make this a private const and add a comment explaining why it is 14

Reply:
Yes, please...  Point to this https://github.com/dotnet/corefx/blob/1df4a4866a90f22f861156b8ff496c24103d39cc/src/Common/src/System/Net/WebSockets/ManagedWebSocket.cs#L61 and this https://github.com/dotnet/corefx/blob/1df4a4866a90f22f861156b8ff496c24103d39cc/src/Common/src/System/Net/WebSockets/ManagedWebSocket.cs#L188

Reply:
Or just to the spec: https://tools.ietf.org/html/rfc6455#section-5.2